aws-android-sdk-sqs-test/src/main/java/com/example/testapp/MainActivity.kt

```diff
@@ -0,0 +1,39 @@
+package com.amazonaws.sqs.test;
+
+import android.os.Bundle
+import android.support.design.widget.Snackbar
+import android.support.v7.app.AppCompatActivity;
+import android.view.Menu
+import android.view.MenuItem
+
+import kotlinx.android.synthetic.main.activity_main.*
+
+class MainActivity : AppCompatActivity() {
```

Comment:
What does this activity do?

Reply:
changed its path name to aws-android-sdk-sqs-test/src/main/java/com/amazonaws/sqs/MainActivity.kt. If we like, we can add UI test here

Reply:
I don't think we need UI tests for this particular service? Please make a not of this in the file.

Reply:
I deleted the file from the project